### PR TITLE
Add support for PlaybackUrl in VideoUploadResult

### PR DESCRIPTION
**Pull request by Suraj3240**
### Brief Summary of Changes
Add support for PlaybackUrl in VideoUploadResult #337

#### What does this PR address?
- [ ] GitHub issue (Add reference - #XX)
- [ ] Refactoring
- [x] New feature
- [ ] Bug fix
- [ ] Adds more tests

#### Are tests included?
- [x] Yes
- [ ] No

#### Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I ran the full test suite before pushing the changes and all the tests pass.

### CODE REVIEW

**Comment by Suraj3240** [comment]
@const-cloudinary

File: CloudinaryDotNet.IntegrationTests/UploadApi/UploadMethodsTest.cs [hunk 1]
```diff
@@ -221,6 +221,7 @@ public void TestUploadLocalVideo()
             Assert.Zero(uploadResult.Pages);
             Assert.Zero(uploadResult.Rotation);
             Assert.NotZero(uploadResult.NbFrames);
+            Assert.IsFalse(string.IsNullOrEmpty(updateResult.PlaybackUrl));
```
Comment (const-cloudinary) [inline CloudinaryDotNet.IntegrationTests/UploadApi/UploadMethodsTest.cs:224]:
```suggestion
            Assert.IsFalse(string.IsNullOrEmpty(uploadResult.PlaybackUrl));
```

**Review by const-cloudinary — CHANGES_REQUESTED** [review]
Please fix a small bug

**Comment by const-cloudinary** [comment]
@Suraj3240 thank you for contribution!

**Comment by Suraj3240** [comment]
@const-cloudinary can you please add the `hacktobefest` label to this

**Comment by colbyfayock** [comment]
hey @Suraj3240 label for the PR aside, the repo has the label so it should qualify for your Hacktoberfest.com contribution

image

**Comment by Suraj3240** [comment]
> hey @Suraj3240 label for the PR aside, the repo has the label so it should qualify for your Hacktoberfest.com contribution
> 
> image

Ok great. Thank You :))

**Pull request merged**